### DISCUSSION TITLE
Prisma Database issue

### DISCUSSION THREAD

**Discussion by prerang** [prisma/prisma#24937] [Q&A]
### Question

Prisma schema loaded from prisma\schema.prisma
Datasource "db": PostgreSQL database "postgres", schema "public" at "aws-0-ap-southeast-1.pooler.supabase.com:5432"

Error: P1001: Can't reach database server at `aws-0-ap-southeast-1.pooler.supabase.com:5432`

Please make sure your database server is running at `aws-0-ap-southeast-1.pooler.supabase.com:5432`.
i am getting this error when i am trying to connect with database

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @prerang 👋 

Do you have both `DATABASE_URL` and `DIRECT_URL` configured? For example:
```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider  = "postgresql"
  url       = env("DATABASE_URL")
  directUrl = env("DIRECT_URL")
}
```
And in your .env file:
```
DATABASE_URL="postgres://postgres:[YOUR PASSWORD]@aws-0-ap-southeast-1.pooler.supabase.com:6543/postgres?pgbouncer=true&connection_limit=1"

DIRECT_URL="postgres://postgres:[YOUR PASSWORD]@aws-0-ap-southeast-1.pooler.supabase.com:5432/postgres"
```

**Reply by prerang** [reply]
Yes I have added everything

**Reply by ludralph** [reply]
Can you verify that that your database is running and not paused? You may also want to try rotating your password and updating the passwords in the environment variables as well.

**Comment by prerang** [comment]
My database is also running and password is also set

**Reply by ludralph** [reply]
Can you share the repository with me? You can DM me the DB URL on Discord so I can take a look as well.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
